README.md

```diff
@@ -16,27 +17,67 @@ For installation, configuration, and details about using the API, check out our
 
 For descriptions of terminology used in APM, take a look at the [official documentation](https://docs.datadoghq.com/tracing/visualization/).
 
+
+## Version Release Lines and Maintenance
+
+| Release Line                                            | Latest Version                                                                                         | Status          |Initial Release | End of Life |
+| :--:                                                    | :--:                                                                                                   | :---:           | :---:          | :---:       |
+| [`v1`](https://github.com/DataDog/dd-trace-js/tree/v1.x) | ![npm v1](https://img.shields.io/npm/v/dd-trace/legacy-v1?color=white&label=%20&style=flat-square)     | **End of Life** | 2021-07-13     | 2022-02-25  |
+| [`v2`](https://github.com/DataDog/dd-trace-js/tree/v2.x) | ![npm v2](https://img.shields.io/npm/v/dd-trace/latest-node12?color=white&label=%20&style=flat-square) | **Maintenance** | 2022-01-28     | 2023-08-15  |
+| [`v3`](https://github.com/DataDog/dd-trace-js/tree/v3.x) | ![npm v3](https://img.shields.io/npm/v/dd-trace/latest?color=white&label=%20&style=flat-square)        | **Current**     | 2022-08-15     | Unknown     |
+
+We currently maintain two release lines, namely `v2` and `v3`.
+Features and bug fixes that are merged are released to the `v3` line and, if appropriate, also the `v2` line.
+
+For any new projects it is recommended to use the `v3` release line:
+
+```sh
+npm install dd-trace
+yarn add dd-trace
+```
+
+However, existing projects that already use the `v2` release line, or projects that need to support Node.js v12 and older, may use the `v2` release line.
+This is done by specifying the version when installing the package:
+
+```sh
+npm install dd-trace@2
+yarn add dd-trace@2
+```
```

Comment:
Should we call out the `latest-node12` tag here? We don't seem to have `latest-nodeN` tags for later versions. Maybe we should?